13-microbiome.Rmd

```diff
@@ -0,0 +1,41 @@
+
+```{r, include = FALSE}
+ottrpal::set_knitr_image_path()
+```
+
+# Microbiome Sequencing
+
+<div class = "warning">
+This chapter is incomplete! If you wish to contribute, please [go to this form](https://forms.gle/dqYgmKH8XXE2ohwD9) or our [GitHub page](https://github.com/fhdsl/Choosing_Genomics_Tools).
+</div>
+
+## Learning Objectives
+
+```{r, fig.alt = "Learning Objectives", out.width = "100%", echo = FALSE}
+ottrpal::include_slide("https://docs.google.com/presentation/d/1YwxXy2rnUgbx_7B7ENH9wpDX-j6JpJz6lGVzOkjo0qY/edit#slide=id.g2668d07d0b9_0_0")
+```
+
```

Comment:
Probably need a title here for this section.

Reply:
I agree! Thank you